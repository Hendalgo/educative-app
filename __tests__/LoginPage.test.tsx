import React from "react";
import renderer from "react-test-renderer";
import Login from "../app/src/pages/LoginPage";
import {expect, describe, it, beforeEach, jest} from "@jest/globals";


describe("Login", () => {
  beforeEach (() => {
    jest.useFakeTimers();
  });
  it("should render the login page", () => {
    const tree = renderer.create(<Login />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});