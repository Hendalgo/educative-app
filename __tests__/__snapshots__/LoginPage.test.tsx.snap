// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login should render the login page 1`] = `
<RCTSafeAreaView
  style={
    {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <View>
    <Text>
      Login
    </Text>
  </View>
</RCTSafeAreaView>
`;

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  style={
    {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <View>
    <Text>
      Login
    </Text>
  </View>
</RCTSafeAreaView>
`;
